inc/admin/class-alias-list-table.php

```diff
@@ -237,8 +237,8 @@ protected function column_domain( $mapping ) {
 	protected function column_active( $mapping ) {
 		$active = $mapping->is_active();
 		if ( $active ) {
-			return esc_html__( 'Active', 'mercator' );
+			return '<span style="color:green;font-weight:bold;">' . esc_html__( 'Active', 'mercator' ) . '</span>';
```

Comment:
Yo, not sure the bold is necessary, also can you use the same colours as the approve/unapprove links from the comments area? Pity those class names are `.approve` and `.unapprove`. `#006505` for active and `#d98500` for inactive. Sorry for the lack of activity here for a while.

Reply:
the css classes are not applicable in this context